modules/ROOT/pages/building-soap-proxy.adoc

```diff
@@ -64,7 +66,7 @@ image:build-soap-proxy-advanced-options.png[SOAP Proxy Advanced Options for Conf
 .. In *Proxy Version*, select `latest`. 
 +
 This value ensures that your API proxy uses the latest released proxy version.
-.. Select the *Validations* checkbox if you want schema validations to be performed on incoming requests.
+.. Select the *Validations* checkbox if you want the content of `body` tag to be validated on incoming requests in addition to the default validations as described in <<SOAP Proxy Schema Validation>>.
```

Comment:
Fix:
```suggestion
.. Select the *Validations* checkbox if you want the content of the `body` tag to be validated on incoming requests in addition to the default validations as described in <<SOAP Proxy Schema Validation>>.
```
Alternative:
```suggestion
.. If you want the content of the `body` tag to be validated (in addition to the default incoming request validations described in <<SOAP Proxy Schema Validation>>), select *Validations*.
```

Reply:
The alternative doesn't quite work because it changes the meaning. I also try to avoid putting things in parenthesis in doc. I can potentially divide it into two sentences, because I agree it is too long and hard to read.